run.sh

```diff
@@ -46,8 +46,8 @@ else
 fi 
 
 docker run --rm -i \
-    -v $(pwd):/tmp/src:z \
-    -v $HOME/.kube:/root/.kube:z \
+    -v $(pwd):/tmp/src \
+    -v $HOME/.kube:/root/.kube \
```

Comment:
is the `:z` problematic? I can't get docker mounts to work correctly without it...

Reply:
@sherl0cks yeah, in my environment it is ... I'll add it back though. Need to troubleshoot more later. We see the same for CASL contributors - it works for some, but not others